src/components/ui/ControlBar/ControlBarItem.tsx

```diff
@@ -44,16 +42,22 @@ export const ControlBarButton: FC<ControlBarButtonProps> = ({
   popOver = null,
   popOverPlacement,
   popOverLabel,
+  tooltipContainerId,
+  tooltipText,
```

Comment:
9 times out of 10, we end up eventually loosening up a prop like this to accept a `ReactNode` along with `string`. It might be worth adding this now.

Reply:
Yeah, good call. I'll update